strimzi/datadog_checks/strimzi/check.py

```diff
@@ -0,0 +1,58 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import copy
+
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheckV2
+from datadog_checks.strimzi.config_models import ConfigMixin
+from datadog_checks.strimzi.metrics import (
+    CLUSTER_OPERATOR_METRICS_MAP,
+    TOPIC_OPERATOR_METRICS_MAP,
+    USER_OPERATOR_METRICS_MAP,
+)
+
+CLUSTER_OPERATOR_NAMESPACE = "strimzi.cluster_operator"
+TOPIC_OPERATOR_NAMESPACE = "strimzi.topic_operator"
+USER_OPERATOR_NAMESPACE = "strimzi.user_operator"
+
+
+class StrimziCheck(OpenMetricsBaseCheckV2, ConfigMixin):
+    DEFAULT_METRIC_LIMIT = 0
+
+    def __init__(self, name, init_config, instances):
+        super(StrimziCheck, self).__init__(name, init_config, instances)
+        self.scraper_configs = []
+        self.check_initializations.appendleft(self.parse_config)
+
+    def parse_config(self):
+        cluster_operator_endpoint = self.instance.get("cluster_operator_endpoint")
+        topic_operator_endpoint = self.instance.get("topic_operator_endpoint")
+        user_operator_endpoint = self.instance.get("user_operator_endpoint")
```

Comment:
Since you are using `ConfigMixin`, you don't need to manually get the instance configs. I.e. for `cluster_operator_endpoint`, it should automatically be set to `self.config.cluster_operator_endpoint`.

Reply:
Note: I think you would need to append `parse_config` [after](https://datadoghq.dev/integrations-core/meta/config-models/#loading) the config mixin check initialization is finished.

Reply:
That's a good idea but the problem is that we need to init the scrapers before other initialisations, ss we have to `appendleft` 🙁